### PR TITLE
Updating main to our current dev release

### PR DESCRIPTION
**Pull request by rmcguire6**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": "latest",
+    "sourceType": "module"
+  },
+  "plugins": ["react", "@typescript-eslint"]
+  "rules": {
```
Comment (Rae-Kwon) [inline .eslintrc.json:16]: There are two rules keys here and on line 21. Maybe remove line 21?

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "extends": ["plugin:react/recommended", "plugin:react/jsx-runtime"],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": "latest",
+    "sourceType": "module"
+  },
+  "plugins": ["react", "@typescript-eslint"]
```
Comment (Rae-Kwon) [inline .eslintrc.json]: Add comma after this line

Comment (JessieFrance) [inline .eslintrc.json]: There should be a comma after this line.

File: .eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  "rules": {
+    "react/jsx-uses-react": "off",
+    "react/react-in-jsx-scope": "off"
+  },
+  "react/jsx-key": [false, { "checkFragmentShorthand": true }]
```
Comment (JessieFrance) [inline .eslintrc.json]: I recommend deleting this line, adding a comma after line 18 in the rules block, and adding this modified version after line 18:
`"react/jsx-key": [0, { "checkFragmentShorthand": true }]`

File: .eslintrc.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  "rules": { "react/react-in-jsx-scope": "off" }
```
Comment (JessieFrance) [inline .eslintrc.json]: This rule looks the same as line 18. I think it can be deleted.

**Review by JessieFrance — APPROVED** [review]
Aside from fixing some minor syntax in `.eslintrc.json`, I think this looks good to merge.

**Review by Rae-Kwon — CHANGES_REQUESTED** [review]
Everything else looks perfect!!

**Review by ArturoGrullon — APPROVED** [review]

**Review by Rae-Kwon — APPROVED** [review]
Looks great!!!

**Pull request merged**